brave/brave-core
# Import chrome extension list

BraveExternalProcessImporterHost will fetch available extensions list
and ask to install them.
Other importing options are done via other utility process but
importing extension should be done in browser process.

NOTE: this commits has many cleanup changes around importing area
that uses deprecated patching style.


Resolves brave/brave-browser#7974

![import_chrome_extensions](https://user-images.githubusercontent.com/6786187/74911456-fcb97580-53ff-11ea-8770-0a265db22f1e.gif)


## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
`npm run test brave_unit_tests -- --filter=ChromeImporterUtilsTest`

Manual test step
1. Launch Chrome and install some extensions from webstore
2. Close Chrome and launch Brave
3. Import from chrome and check extensions option
4. After seeing import done dialog, wait and check extensions are installed.
## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
